CMakeLists.txt

```diff
@@ -237,15 +223,20 @@ MACRO(CheckAndAddFlag flag)
 	unset(checkedFlag)
 ENDMACRO()
 
+OPTION(ENABLE_WERROR "Build with warnings turned into errors" OFF)
```

Comment:
please change Jenkinsfile, .travis.yml  and appveyor.yml so the ci builds are run with WError